docs/source/getting-started/Setting-up-hypervisor-driver.adoc

```diff
@@ -1,20 +1,20 @@
-[[docker-machine-driver-install]]
-= Installing Docker Machine drivers
+[[set-up-hypervisor-driver]]
```

Comment:
regarding the filename, let's stick to all lowercase. In this case we should not even rename the file, just saying.